### PR TITLE
Fix the incorrect histogram example in metrics.json

### PR DESCRIPTION
**Pull request by songy23**
There are 2 buckets (-inf, 1), [1, +inf) but 3 bucket counts.

### CODE REVIEW

File: examples/metrics.json [hunk 1]
```diff
@@ -81,11 +81,11 @@
...
-                    "count": 3,
-                    "sum": 3,
-                    "bucketCounts": [1,1,1],
+                    "count": 2,
+                    "sum": 2,
+                    "bucketCounts": [1,1],
                     "explicitBounds": [1],
-                    "min": 1,
+                    "min": 0,
                     "max": 1,
```
Comment (jack-berg) [inline examples/metrics.json:89]: If the count is 2, and the min is zero, then one of the two meaurements is zero. Therefore for the sum to be 2, the other measurement must be 2.

Comment (songy23) [reply examples/metrics.json]: Good catch, fixed.

**Review by jack-berg — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
@open-telemetry/specs-metrics-approvers please review.

**Review by MrAlias — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

**Review by songy23 — COMMENTED** [review]
Thanks for the reviews, this should be good to go

**Pull request merged**